internal/handlers/pg/msg_update.go

```diff
@@ -63,7 +63,7 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
-	created, err := h.pgPool.EnsureTableExist(ctx, sp.db, sp.collection)
+	created, err := h.pgPool.CreateTableIfNotExists(ctx, sp.db, sp.collection)
```

Comment:
Just a func rename. This method needs some special handling to deal with `request` correctly, but it's not in scope of this PR.